Riot/Modules/TabBar/TabBarCoordinator.swift

```diff
@@ -705,6 +721,128 @@ final class TabBarCoordinator: NSObject, TabBarCoordinatorType {
     }
 }
 
+// Tchap: Manage e-mail invitation
+extension TabBarCoordinator {
+    private func promptUserToFillAnEmailToInvite(completion: @escaping ((String) -> Void)) {
+        currentAlertController?.dismiss(animated: false)
+        
+        let alertController = UIAlertController(title: TchapL10n.contactsInviteByEmailTitle,
+                                                message: TchapL10n.contactsInviteByEmailMessage,
+                                                preferredStyle: .alert)
+        
+        // Add textField
+        alertController.addTextField(configurationHandler: { textField in
+            textField.isSecureTextEntry = false
+            textField.placeholder = nil
+            textField.keyboardType = .emailAddress
+        })
+        
+        // Cancel action
+        let cancelAction = UIAlertAction(title: TchapL10n.actionCancel,
+                                         style: .cancel) { [weak self] _ in
+            self?.currentAlertController = nil
+        }
+        alertController.addAction(cancelAction)
+        
+        // Invite action
+        let inviteAction = UIAlertAction(title: TchapL10n.actionInvite,
```

Comment:
I realized here we should use the current VectorL10n.invite string

Reply:
Fixed in last commit